### PR TITLE
feat: introduce github.com/einride/spanner-aip-go tool

### PR DESCRIPTION
**Pull request by alethenorio**

### CODE REVIEW

File: tools/sgspanneraipgo/tools.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package main
```
Comment (viktorvoltaire) [inline tools/sgspanneraipgo/tools.go:1]: Should be sgspanneraipgo

File: tools/sgspanneraipgo/tools.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	version = "0.39.0"
+	name    = "spanner-aip-go"
+)
+
+type SpannerAIP sg.Namespace
```
[the remark this answers is not in the document]
Comment (alethenorio) [reply tools/sgspanneraipgo/tools.go:20]: Good point. I am not done with the PR just yet. Have a few more things to fix but ran out of time. Will fix it

**Review by viktorvoltaire — APPROVED** [review]
Awesome!

**Comment by alethenorio** [comment]
I have fixed the issues. I had planned on looking into adding some Prepare command using a go.mod file since that is likely to be as common (so the tool version matches whatever library the user is using locally) however trying to get that to play together with the `Command` target in some coherent way, along with the fact that the target became so small I think it is best handled locally for now until we find a better solution

**Pull request merged**